blockchain/upgrade.go

```diff
@@ -1025,6 +1034,31 @@ func upgradeToVersion6(ctx context.Context, db database.DB, chainParams *chaincf
 	return nil
 }
 
+// upgradeToVersion7 upgrades a version 6 blockchain database to version 7.
+func upgradeToVersion7(db database.DB, chainParams *chaincfg.Params, dbInfo *databaseInfo) error {
+	log.Info("Adding treasury databases...")
+	start := time.Now()
+
+	// Add treasury database.
+	err := addTreasuryBucket(db)
+	if err != nil {
+		return err
+	}
+
+	// Add tspend database.
+	err = addTSpendBucket(db)
+	if err != nil {
+		return err
+	}
+
+	// XXX Rescan blocks for TADD/TSPEND
```

Comment:
This obviously needs to be completed.

Reply:
As debated, no need for upgrade because hardfork.